generator/types.go

```diff
@@ -815,7 +816,7 @@ func (t *typeResolver) ResolveSchema(schema *spec.Schema, isAnonymous, isRequire
 	extType, isExternalType := t.resolveExternalType(schema.Extensions)
 	if isExternalType {
 		tpe, pkg, alias := t.knownDefGoType(t.ModelName, *schema, t.goTypeName)
-		debugLog("found type %s declared as external, imported from %s as %s. Has type hints? %t, rendered has embedded? %t",
+		log.Printf("found type %s declared as external, imported from %s as %s. Has type hints? %t, rendered has embedded? %t",
```

Comment:
can you put this back to debugLog ?

You can see logs with the env var `DEBUG=1` or `SWAGGER_DEBUG=`